internal/convert/process_config.go

```diff
@@ -257,6 +288,9 @@ func newMongosProcessConfig(p *opsmngr.Process) *ProcessConfig {
 	if p.Args26.Security != nil {
 		pc.Security = p.Args26.Security
 	}
+	if p.Args26.Security != nil {
+		pc.Security = p.Args26.Security
```

Comment:
side note, Args26 is not the catchiest name is it? 😅

Reply:
no, and this looks like copy/paste, need to remove this block